### DISCUSSION TITLE
[Feature request] Create your own custom charts on Sekai Viewer like Bestdori!

### DISCUSSION THREAD

**Discussion by BentleyBentayga** [Sekai-World/sekai-viewer#355] [Ideas]
**Motivation.**
If you downloaded some plugins, unblocked it, put to the plugins folder, run Ched, and try to execute a plugin, it'll cause an exception of both plugins, the import failed and the plugin texture not showing, so you can try Custom Charts on Sekai Viewer, it is an alternative to Ched.

You could see the "Charts" tab, and then click "Publish chart", there is lots of community made ProSeka charts. No file needed, you can create your own chart just like Bestdori!.
![sugg](https://user-images.githubusercontent.com/94912171/146193392-801b851d-784e-495d-bba1-09e3663a3e16.png)

**Comment by dnaroma** [comment]
This will need a chart editor in browser

**Reply by BentleyBentayga** [reply]
it'll be better, easier, and more handier than using Ched.

**Reply by dnaroma** [reply]
> it'll be better, easier, and more handier than using Ched.

Do you already have one ready for use?

**Reply by BentleyBentayga** [reply]
hmmm.... maybe i already have ched, but i'll use the chart maker from the Sekai Viewer if it was added.
